This is to avoid a very weird compiler bug
on -stable. Certain combinations of -O#
and -march were making kicq core dump.

--- configure.orig	Mon Aug 14 23:25:24 2000
+++ configure	Mon Aug 14 23:26:28 2000
@@ -6406,7 +6406,7 @@
 s%@SHELL@%$SHELL%g
 s%@CFLAGS@%$CFLAGS%g
 s%@CPPFLAGS@%$CPPFLAGS%g
-s%@CXXFLAGS@%$CXXFLAGS%g
+s%@CXXFLAGS@%%g
 s%@FFLAGS@%$FFLAGS%g
 s%@DEFS@%$DEFS%g
 s%@LDFLAGS@%$LDFLAGS%g
